webui/react/src/routes/routes.ts

```diff
@@ -113,21 +125,41 @@ const routes: RouteConfig[] = [
     icon: 'cluster',
     id: 'cluster',
     needAuth: true,
-    path: '/cluster/:tab?',
+    path: '/cluster',
+    redirect: '/clusters',
+    title: 'Cluster',
+  },
+  {
+    icon: 'cluster',
+    id: 'cluster',
+    needAuth: true,
+    path: '/cluster/:tab',
     redirect: '/clusters',
     title: 'Cluster',
   },
   {
     icon: 'cluster',
     id: 'clusters',
     needAuth: true,
-    path: '/clusters/:tab?',
+    path: '/clusters',
```

Comment:
**question:** huh, is this something that changed with the version bump or was it always like this? I remember if we didn't put the more specific path before the general path in the array, it would resolve to the general before the more specific route, but playing with the preview, it doesn't seem to be the case?

Reply:
It comes from [this](https://github.com/remix-run/react-router/discussions/8753)
<img width="1392" alt="image" src="https://user-images.githubusercontent.com/109113616/180822015-c413b42b-8a6a-4a16-9e2c-64822fe9b9db.png">
`react-router-dom v6` doesnt support this syntax so I changed it

Reply:
it's a bit more verbose but it's cleaner IMO